app-misc/asciinema/patches/asciinema-2.0.1.patchset

```diff
@@ -0,0 +1,53 @@
+From 0eb3de49de7707d3a52aab86a0d6f2b44fdc11a3 Mon Sep 17 00:00:00 2001
+From: Crestwave <crest.wave@yahoo.com>
+Date: Fri, 2 Nov 2018 07:55:00 +0000
+Subject: [PATCH 1/1] Add long_description, data_files and test_requires to
```

Comment:
@Crestwave, that's not very important, but there is a tiny difference between a patchset created with git and one created with `hp --extract-patchset asciinema` (or `hp -e asciinema`):

```diff
-Subject: [PATCH 1/1] Add long_description, data_files and test_requires to
- setup.py
+Subject: Add long_description, data_files and test_requires to setup.py
 
----
- setup.py | 14 ++++++++++++++
- 1 file changed, 14 insertions(+)

  diff --git a/setup.py b/setup.py
```

So, since the preferred method to create a patchset for for haikuports is to use haikuporter's `--extract-patchset` feature, would you like to try that?
The advantages are that the patchset is later modified the diff will be smaller because the `[PATCH 1/1]` string will already have been removed, as well as the short list of files changed.
The steps to create the patchset:
```
hp -c asciinema
hp -B asciinema
cd /boot/home/haikuports/dev-python/asciinema/work-2.0.1/sources/asciinema-2.0.1
git log # if you want to check something, and maybe use other git commands alter the result of the patchset generation
hp -e asciinema
```

Reply:
Thanks! So are you always supposed to use patchsets or are there specific usecases for regular `.patch` files made with `diff`, like if it's only to do one thing?